oiosaml/src/test/java/dk/gov/oio/saml/util/StringUtilTest.java

```diff
@@ -0,0 +1,46 @@
+package dk.gov.oio.saml.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.opensaml.core.xml.util.XMLObjectSupport;
+import org.opensaml.messaging.context.MessageContext;
+import org.opensaml.saml.common.SAMLObject;
+import org.w3c.dom.*;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.UUID;
+
+class StringUtilTest {
```

Comment:
I miss comments and better naming of the tests.
I know that they are simple, but the quality here is poor.